src/Map/assets/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "@symfony/ux-map",
+    "description": "Integrates interactive maps in your Symfony applications",
+    "license": "MIT",
+    "version": "1.0.0",
+    "symfony": {
+        "importmap": {
+            "@hotwired/stimulus": "^3.0.0",
+            "@symfony/ux-map/abstract-map-controller": "path:%PACKAGE%/dist/abstract_map_controller.js"
+        }
+    },
+    "peerDependencies": {
+        "@hotwired/stimulus": "^3.0.0"
```

Comment:
Can we start enforcing >= 3.2 ?

Reply:
Yes, but I simply took the same configuration than other packages:

<img width="1234" alt="image" src="https://github.com/user-attachments/assets/f4d202ad-f3ef-47d7-9581-573eca812920">